app/assets/templates/frame_template.html

```diff
@@ -3,7 +3,7 @@
     <div class="widget_title panel-heading" ng-mouseenter="showControls = true" ng-mouseleave="showControls = false">
       {{getTitle()}}
       <div class="btn-group pull-right graph_config_menu" ng-show="showControls || showTab">
-        <div class="btn btn-primary btn-sm" ng-click="refreshFrame()" tooltip-append-to-body="true" tooltip="Refresh"><i ng-class="{spin:requestsInFlight}" class="icon-cycle"></i></div>
+        <div class="btn btn-primary btn-sm" ng-click="refreshFrame()" tooltip-append-to-body="true" tooltip="Refresh"><i ng-class="{spin:requestsInFlight}" class="glyphicon glyphicon-refresh"></i></div>
```

Comment:
There are a couple more icons which look way better in Glyphicons - like the permalink icon and the trash bin. But let's keep that for a future PR :)